samsawan/SendBird-JavaScript
# PL-13 : Remove Sound From Widget

# Things Done
- Removed code referencing bad mp3 sound
- Removed code calling the function that played the sound
- Sorted the imports on widget.js to match PL-4's PR.

# How To Test
- Check if the sound doesn't come out of the speakers.
- Check nothing else is broken.
-

# Related PRs
